add warning on old custom dashboards and rephrase existing page (#17214)
netdata/netdata: src/web/gui/custom/README.md

# Build new dashboards with HTML

> >❗This feature is only available on v1 dashboards, it hasn't been port-forwarded to v2.

Reviewer: ```suggestion
>❗This feature is only available on v1 dashboards and is not currently supported on v2.
```

I am not sure "port-forwarded" is the correct term.

>  For more information on the new Custom Dashboards check [this documentation](https://github.com/netdata/netdata/blob/master/docs/cloud/visualize/dashboards.md).

You can build dashboards with Netdata charts just with some basic HTML knowledge.

> These dashboards allow you to  utilize any or all of the available chart libraries, on the same dashboard.

Reviewer: ```suggestion
These dashboards allow you to utilize any or all of the available chart libraries, on the same dashboard.
```
